### ISSUE TITLE
azure-functions-core-tools@3.0.3528: hash check failed

### ISSUE THREAD

**Issue by rohancragg** [ScoopInstaller/Main#2174]
```
Updating 'azure-functions-core-tools' (3.0.3477 -> 3.0.3528)
Downloading new version
Starting download with aria2 ...
Invoke-Expression: C:\Users\rohan\scoop\apps\scoop\current\lib\install.ps1:287
Line |
 287 |          Invoke-Expression $aria2 | ForEach-Object {
     |          ~~~~~~~~~~~~~~~~~~~~~~~~
     | This script contains malicious content and has been blocked by your antivirus software.


Checking hash of Azure.Functions.Cli.win-x64.3.0.3528.zip ... Get-FileHash: C:\Users\rohan\scoop\apps\scoop\current\lib\install.ps1:691
Line |
 691 |  …          return (Get-FileHash -Path $file -Algorithm $algname).Hash.T …
     |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path
     | 'C:\Users\rohan\scoop\cache\azure-functions-core-tools#3.0.3528#https_github.com_Azure_azure-functions-core-tools_releases_download_3.0.3528_Azure.Functions.Cli.win-x64.3.0.3528.zip_dl.7z' because it does not exist.

ERROR You cannot call a method on a null-valued expression.
ERROR Hash check failed!
App:         main/azure-functions-core-tools
URL:         https://github.com/Azure/azure-functions-core-tools/releases/download/3.0.3528/Azure.Functions.Cli.win-x64.3.0.3528.zip#/dl.7z
Expected:    bfd0359b912c498f70d34620bfa2712735fa1482c9d453edcbd7c65f98ebe534
Actual:

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=azure-functions-core-tools%403.0.3528%3a+hash+check+failed
```

**Comment by rohancragg** [comment]
Steps do not resolve the issue and I cannot seem to re-open

**Comment by rohancragg** [comment]
```
> scoop config aria2-enabled false
'aria2-enabled' has been set to 'false'
```
Second attempt:
```
> scoop update; scoop uninstall azure-functions-core-tools; scoop install azure-functions-core-tools
Updating Scoop...
Updating 'extras' bucket...
Updating 'instrumenta' bucket...
Updating 'main' bucket...
Updating 'nerd-fonts' bucket...
Updating 'Sysinternals' bucket...
Updating 'versions' bucket...
Scoop was updated successfully!
Uninstalling 'azure-functions-core-tools' ().
'azure-functions-core-tools' was uninstalled.
Installing 'azure-functions-core-tools' (3.0.3528) [64bit]
Azure.Functions.Cli.win-x64.3.0.3528.zip (270.0 MB) [===============================================================>                               ]  67%
The response ended prematurely, with at least 91917899 additional bytes expected.
URL https://github.com/Azure/azure-functions-core-tools/releases/download/3.0.3528/Azure.Functions.Cli.win-x64.3.0.3528.zip#/dl.7z is not valid
```

Third attempt succeeded so issue was probably a network error as suggested.
